src/main/java/com/rackspace/salus/policy/manage/entities/TenantMetadata.java

```
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.policy.manage.entities;
+
+import com.rackspace.salus.policy.manage.web.model.TenantMetadataDTO;
+import com.vladmihalcea.hibernate.type.json.JsonStringType;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.validator.constraints.NotBlank;
+
+@Entity
+@Table(name = "tenant_metadata", indexes = {
+    @Index(name = "by_tenant", columnList = "tenant_id")
```

I'm never quite sure if `@Index` needs the DB column name `tenant_id` or the JPA field name `tenantId`, so heads up this may or may not work with `tenant_id`.

---

From what I've seen I think it's the column name.  I assume I can see that when I run everything in dev.  I'll try remember to have a look later.

---

Yeah, it'll fail fast if it's not the right one of the two choices.